django/db/models/functions.py

```diff
@@ -137,3 +137,16 @@ class Upper(Func):
 
     def __init__(self, expression, **extra):
         super(Upper, self).__init__(expression, **extra)
+
+
+class Now(Func):
+    template = 'CURRENT_TIMESTAMP()'
```

Comment:
What about defaulting to `NOW()` instead? This should be compatible with PostgreSQL and MySQL and that's what I would expect a `Now()` function to be doing.